frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.java

```diff
@@ -946,7 +953,11 @@ private boolean restoreFromBundle(@Nullable Bundle savedInstanceState, @Nullable
 
             //Successfully restored state
             return true;
-        } catch (JSONException e) {
+        } catch (Throwable ex) {
```

Comment:
Here I went back to Throwable because we experienced a lot of
java.lang.IndexOutOfBoundsException
Index: 1, Size: 1